.travis.yml

```diff
@@ -11,22 +11,30 @@ matrix:
   allow_failures:
     - env: SHELL=zsh
 script:
-  - >
-    if ${LINT:-false}; then
-      shellcheck -s sh bin/shpec  # Just use th `sh` dialect
-    else
-      time $SHELL bin/shpec shpec/shpec_shpec.sh
-    fi
+  - |
+    ${LINT:-false} &&
+      exec shellcheck -s sh bin/shpec  # `sh` refers to POSIX
+    $SHELL bin/shpec shpec/shpec_shpec.sh
```

Comment:
hey @hlangeveld - i don't understand (which is why i had this in an `if / else` block) - why doesn't travis run the shpec tests in the build matrix for your code when `LINT=true`?
(https://travis-ci.org/rylnd/shpec/jobs/60934292  ... hmmm and it might be nice if `shellcheck` output *something* when it ran successfully  - i don't see it has a `--verbose` or `--trace` option though.)